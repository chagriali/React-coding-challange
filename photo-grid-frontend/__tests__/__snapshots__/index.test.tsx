import { render, screen } from "@testing-library/react";
import Home from "@/pages/index";

describe("index", () => {
  it("Should render correctly", () => {
    const home = render(<Home />);
    expect(home).toMatchSnapshot();
  });
});
