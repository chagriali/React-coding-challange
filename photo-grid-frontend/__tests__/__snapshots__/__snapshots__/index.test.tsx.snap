// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index Should render correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex flex-col h-screen justify-between"
      >
        <nav
          class="relative flex flex-wrap items-center justify-between px-2 py-3 bg-blue-500 mb-3"
        >
          <div
            class="container px-4 mx-auto flex flex-wrap items-center justify-between"
          >
            <div
              class="w-full relative flex justify-between lg:w-auto lg:static lg:block lg:justify-start"
            >
              <a
                class="text-sm font-bold leading-relaxed inline-block mr-4 py-2 whitespace-nowrap uppercase text-white"
                href="#pablo"
              >
                React coding challenge
              </a>
            </div>
          </div>
        </nav>
        <footer
          class="p-4 bg-white rounded-lg shadow md:flex md:items-center md:justify-between md:p-6 dark:bg-gray-800"
        >
          <span
            class="text-sm text-gray-500 sm:text-center dark:text-gray-400"
          >
            © 2022 . All Rights Reserved.
          </span>
          <ul
            class="flex flex-wrap items-center mt-3 text-sm text-gray-500 dark:text-gray-400 sm:mt-0"
          >
            <li>
              <a
                class="mr-4 hover:underline md:mr-6 "
                href="#"
              >
                About
              </a>
            </li>
            <li>
              <a
                class="hover:underline"
                href="#"
              >
                Contact
              </a>
            </li>
          </ul>
        </footer>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex flex-col h-screen justify-between"
    >
      <nav
        class="relative flex flex-wrap items-center justify-between px-2 py-3 bg-blue-500 mb-3"
      >
        <div
          class="container px-4 mx-auto flex flex-wrap items-center justify-between"
        >
          <div
            class="w-full relative flex justify-between lg:w-auto lg:static lg:block lg:justify-start"
          >
            <a
              class="text-sm font-bold leading-relaxed inline-block mr-4 py-2 whitespace-nowrap uppercase text-white"
              href="#pablo"
            >
              React coding challenge
            </a>
          </div>
        </div>
      </nav>
      <footer
        class="p-4 bg-white rounded-lg shadow md:flex md:items-center md:justify-between md:p-6 dark:bg-gray-800"
      >
        <span
          class="text-sm text-gray-500 sm:text-center dark:text-gray-400"
        >
          © 2022 . All Rights Reserved.
        </span>
        <ul
          class="flex flex-wrap items-center mt-3 text-sm text-gray-500 dark:text-gray-400 sm:mt-0"
        >
          <li>
            <a
              class="mr-4 hover:underline md:mr-6 "
              href="#"
            >
              About
            </a>
          </li>
          <li>
            <a
              class="hover:underline"
              href="#"
            >
              Contact
            </a>
          </li>
        </ul>
      </footer>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
